spec/features/saml/loa3_sso_spec.rb

```diff
@@ -5,36 +5,61 @@
   include IdvHelper
 
   context 'First time registration' do
-    it 'redirects to original SAML Authn Request after IdV is complete', email: true do
+    let(:email) { 'test@test.com' }
+    before do
       allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
-      saml_authn_request = auth_request.create(loa3_with_bundle_saml_settings)
+      @saml_authn_request = auth_request.create(loa3_with_bundle_saml_settings)
+    end
+
+    it 'redirects to original SAML Authn Request after IdV is complete', email: true do
       xmldoc = SamlResponseDoc.new('feature', 'response_assertion')
-      email = 'test@test.com'
 
-      visit saml_authn_request
-      click_link t('sign_up.registrations.create_account')
-      submit_form_with_valid_email
-      click_confirmation_link_in_email(email)
-      submit_form_with_valid_password
-      set_up_2fa_with_valid_phone
-      enter_2fa_code
+      visit @saml_authn_request
+
+      saml_register_loa3_user(email)
 
       expect(current_path).to eq verify_path
-      click_on 'Yes'
+
+      click_idv_begin
+
       user = User.find_with_email(email)
       complete_idv_profile_ok(user.reload)
       click_acknowledge_personal_key
 
       expect(page).to have_content t('titles.loa3_verified.true', app: APP_NAME)
       click_on I18n.t('forms.buttons.continue_to', sp: 'Test SP')
-      expect(current_url).to eq saml_authn_request
+      expect(current_url).to eq @saml_authn_request
 
       user_access_key = user.unlock_user_access_key(Features::SessionHelper::VALID_PASSWORD)
       profile_phone = user.active_profile.decrypt_pii(user_access_key).phone
 
       expect(xmldoc.phone_number.children.children.to_s).to eq(profile_phone)
     end
 
+    it 'allows the user to select verification via USPS letter', email: true do
+      visit @saml_authn_request
+
+      saml_register_loa3_user(email)
+
+      click_idv_begin
+
+      fill_out_idv_form_ok
+      click_idv_continue
+      fill_out_financial_form_ok
+      click_idv_continue
+
+      click_idv_address_choose_usps
+
+      click_on t('idv.buttons.send_letter')
+
+      expect(current_path).to eq verify_review_path
+
+      fill_in :user_password, with: user_password
+      click_submit_default
+
+      expect(current_url).to eq verify_confirmations_url
```

Comment:
Sure, I get that, but they are still able to do more on the site, right? Once on the `verify/confirmations` page, they will acknowledge their personal key, and after that, they should be sent to the `account` page, where they should see a message about needing to confirm their USPS code.

I suggest we merge https://github.com/18F/identity-idp/pull/1424 (I approved it last night. It just needs to be squashed), then rebase this PR against master so we can properly test the whole flow as the user would experience it.